python/nav/eventengine/alerts.py

```diff
@@ -133,6 +134,8 @@ def post_alert(self, history=None):
         alert.history = history
         alert.save()
         self._post_alert_messages(alert)
+        if export.exporter:
+            export.exporter.export(alert)
```

Comment:
Haha, FactoryFactory.Factory.factorybuilder :)

Reply:
I was thinking the same thing as I wrote it out :stuck_out_tongue: